src/widget/wtracktableview.cpp

```diff
@@ -567,6 +573,11 @@ void WTrackTableView::createActions() {
             this, [this] { slotScaleBpm(Beats::FOURTHIRDS); });
     connect(m_pBpmThreeHalvesAction, &QAction::triggered,
             this, [this] { slotScaleBpm(Beats::THREEHALVES); });
+
+    m_pColorPickerAction = new WColorPickerAction(this);
+    m_pColorPickerAction->colorPicker()->setObjectName("TrackColorPicker");
+    connect(m_pColorPickerAction->colorPicker(), &WColorPicker::colorPicked,
```

Comment:
We should think how to move the connect() operation into WColorPickerAction and avoid to publicly expose a pointer to the child object.